src/Catalog.php

```diff
@@ -15,14 +15,16 @@
  * under the License.
  */
 
-require_once(dirname(__FILE__) . '/gen/killbill_catalog_attributes_simple.php');
+namespace Killbill\Client;
 
-class Killbill_Catalog extends Killbill_CatalogAttributesSimple {
+use Killbill\Client\Type\CatalogAttributes;
+
+class Catalog extends CatalogAttributes {
     private $fullCatalog;
 
     public function initialize($headers = null) {
-        $response = $this->_get(Killbill_Client::PATH_CATALOG, $headers);
-        $this->fullCatalog = json_decode($response->body);
+        $response = $this->_get(Client::PATH_CATALOG, $headers);
+        $this->fullCatalog = json_decode($response->body)[0];
```

Comment:
In case of multi-versioned catalogs, multiple versions will be [returned](https://github.com/killbill/killbill/blob/master/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/CatalogResource.java#L137-L139) (so we need to keep the array).